addons/OPT/REVIVE/fn_clientInitEH.sqf

```diff
@@ -115,21 +114,25 @@ DFUNC(playerHandleDamage) =
 	GVAR(playerHandleDamage_projectile) = _projectile; 
 	GVAR(playerHandleDamage_damage) = _damage; 
 
-	// Schade hoeher eingestellen Werte Blocken um Autorespwan zu verhindern.
-	if (_damage >= GVAR(MAX_DAMAGE)) then 
-    { 
-		_unit setDamage GVAR(MAX_DAMAGE);		
-	};	
+	private _resultingDamage = _damage;
 
-	if (getDammage _unit >= GVAR(MAX_DAMAGE)) then 
-    { 
-		_unit setDamage GVAR(MAX_DAMAGE);		
-	};	
+	if (_damage >= GVAR(MAX_DAMAGE)) then {   
+		[FUNC(playercheckINCAPACITATED), 1,""] call CLib_fnc_wait;
+		_resultingDamage = 0; 
```

Comment:
Das hier dürfte den Effekt haben, dass der Spieler komplett geheilt wird :eyes:

Reply:
Was egal sein dürfte, da er durch den Revive die Health neu gesetzt wird.

Werte > 0 machen leider Probleme (besonders ab 0.75)

Reply:
Dann sollte hier ein Kommentar mit genau dieser Info hin :)

Reply:
```
		// Making him invulnerable to prevent forced respawn and random damage that accumulates, if he get overkilled
		// the Revive-Function should set the desired damage after reviving the body.
```